homeassistant/components/recorder/models.py

```diff
@@ -0,0 +1,115 @@
+"""Models for SQLAlchemy."""
+
+import json
+from datetime import datetime
+
+from sqlalchemy import (Boolean, Column, DateTime, ForeignKey, Index, Integer,
+                        String, Text, distinct)
+from sqlalchemy.ext.declarative import declarative_base
+import homeassistant.util.dt as dt_util
+from homeassistant.remote import JSONEncoder
+
+# SQLAlchemy Schema
+# pylint: disable=invalid-name
+Base = declarative_base()
+
+
+class Events(Base):
+    # pylint: disable=too-few-public-methods
+    """Event history data."""
+
+    __tablename__ = 'events'
+    event_id = Column(Integer, primary_key=True)
+    event_type = Column(String(32), index=True)
+    event_data = Column(Text)
```

Comment:
Should this be [JSON type](http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.JSON) ?

This blog post makes it seem like it is possible for SQLite (probably just be a string): http://variable-scope.com/posts/creating-a-json-column-type-for-sqlalchemy